### PR TITLE
New: Report and cancel scan on unhandled errors

### LINKED ISSUE #1684: [extension-browser] Detect and recover from analysis failures [1.5]
If for some reason analysis fails, we can hang the UI waiting for results. While each of these represent a bug we'll want to fix, we should also have a failsafe mechanism that detects this state and/or cancels the analysis after a certain amount of time.

### PR DESCRIPTION
**Pull request by antross**
Fix #1684

## Pull request checklist

Make sure you:

- [x] Signed the [Contributor License Agreement](https://cla.js.foundation/webhintio/hint)
- [x] Followed the [commit message guidelines](https://webhint.io/docs/contributor-guide/getting-started/pull-requests/#commit-messages)

For non-trivial changes, please make sure you also:

- [ ] Added/Updated related documentation.
- [ ] Added/Updated related tests.

### CODE REVIEW

File: packages/extension-browser/src/devtools/views/pages/error.tsx [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+
+import { ErrorData } from '../../../shared/types';
+
+import { getMessage } from '../../utils/i18n';
+
+import Page from '../page';
+
+import * as styles from './error.css';
+
+const openIssueUrl = 'https://github.com/webhintio/hint/issues/new?labels=type%3Abug&amp;template=1-bug-report.md&amp;title=%5BBug%5D+Bug+description';
```
Comment (molant) [inline packages/extension-browser/src/devtools/views/pages/error.tsx]: I think we should create a template for the browser extension. What do you think?

Comment (antross) [reply packages/extension-browser/src/devtools/views/pages/error.tsx]: I like that - I'll draft one up as part of this PR.

Comment (antross) [reply packages/extension-browser/src/devtools/views/pages/error.tsx]: Done!

File: .github/ISSUE_TEMPLATE/2-bug-report-browser.md [hunk 2]
```diff
@@ -0,0 +1,71 @@
...
+Ignored resources:
+* [x] None
+* [ ] Different origin
+* [ ] Custom: <!-- ✍️ -->
+
+</details>
+
+## Debug output
+
+Please include the raw output generated by `webhint`.
```
Comment (molant) [inline .github/ISSUE_TEMPLATE/2-bug-report-browser.md]: Do we have a debug output in the extension?

Comment (antross) [reply .github/ISSUE_TEMPLATE/2-bug-report-browser.md]: No. I was thinking of a copy/paste or screenshot of whatever got displayed. Suggestions on better wording?

**Comment by antross** [comment]
@sarvaje Can you take a look at this PR when you get the chance? Thanks!

File: .github/ISSUE_TEMPLATE/2-bug-report-browser.md (hunk 2 above)
Comment (molant) [reply .github/ISSUE_TEMPLATE/2-bug-report-browser.md]: What about

```suggestion
Please include the content of the `Error details` section if an error message was displayed.
```

File: .github/ISSUE_TEMPLATE/2-bug-report-browser.md [hunk 3]
```diff
@@ -0,0 +1,71 @@
...
+
+<details>
+<summary>webhint’s raw output</summary>
```
Comment (molant) [inline .github/ISSUE_TEMPLATE/2-bug-report-browser.md]: If we go with the above, then this should be something like

```suggestion
<summary>webhint’s Error details</summary>
```

Comment (antross) [reply .github/ISSUE_TEMPLATE/2-bug-report-browser.md]: Done!

**Pull request merged**